site/src/scaffold/scripts/popover.ts

```diff
@@ -0,0 +1,72 @@
+export function initPopovers(container: HTMLElement) {
+	container.addEventListener('click', (event: Event) => {
+		let targetPopover: HTMLDetailsElement | null;
```

Comment:
I may be reading this incorrectly, but it seems to me like we can condense these two lines without sacrificing readability, thanks to our friend the `?.` optional chaining operator. 

```ts
	const targetPopover =
			event.target instanceof Element &&
			(event.target.closest('details.popover') as HTMLDetailsElement) ||
			// or
			event.target instanceof Element &&
			event.target.shadowRoot?.activeElement?.closest('details.popover') as HTMLDetailsElement;
```

Reply:
Thanks for the suggestion, updated!